apps/client/src/components/add-toast/add-toasts.tsx

```diff
@@ -36,17 +59,157 @@ export const AddToast: React.FC<Props> = ({
       width: '37.5rem',
     },
   };
+  const dialogContentStyle = {
+    direction: 'rtl',
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column',
+    gap: '1rem',
+    color: 'var(--white-color)',
+    '&.MuiDialogContent-root': {
+      height: '80%',
+    },
+  };
+
+  const buttonStyles = {
+    fontSize: '1rem',
+    width: '90%',
+    color: 'black',
+    transitionDuration: '100ms',
+    '&:hover': {
+      cursor: 'pointer',
+      transform: 'scale(1.2)',
+    },
+  };
 
   const handleClickAway = () => {
     setIsAddToastDialogOpe(false);
   };
 
+  const [solidsPick, setSolidsPick] = useState<string[]>([]);
+
+  const handleChange = (event: SelectChangeEvent<typeof solidsPick>) => {
+    const {
+      target: { value },
+    } = event;
+    setSolidsPick(typeof value === 'string' ? value.split(',') : value);
+  };
+
+  const solids = [
+    'פלאפל',
+    'פיצות',
+    '',
```

Comment:
Was it intended for there to be an empty string here?

Reply:
it meant to be a divider which later will be disabled